include/odp/api/spec/reassembly.h

```diff
@@ -0,0 +1,84 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * @file
+ *
+ * ODP REASSEMBLY API
+ */
+
+#ifndef ODP_API_SPEC_REASSEMBLY_H_
+#define ODP_API_SPEC_REASSEMBLY_H_
+#include <odp/visibility_begin.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @defgroup odp_reassembly ODP REASSEMBLY
+ *  Reassembly
+ *  @{
+ */
+
+/**
+ * Fragmentation reassembly layer configuration
+ *
+ */
+typedef struct odp_reassembly_layer_config_t {
+	/** Attempt inline reassembly of packets */
+	odp_bool_t enable;
+
+	/** Maximum wait time in ns, the fragment would wait in the reassembly
+	 *  offload for the arrival of further fragments. */
+	uint64_t max_wait_time;
+
+	/** Maximum number of fragments that can be reassembled */
+	uint16_t max_nb_frags;
```

Comment:
This should be max_num_frags to be consistent with other parts of the API.

Also, default values should be specified for all the fields.

Reply:
> This should be max_num_frags to be consistent with other parts of the API.

Agreed.

> Also, default values should be specified for all the fields.

Just curious, what would default value signify? This is a config that application would pass, right?

Reply:
> Just curious, what would default value signify? This is a config that application would pass, right?

odp_pktio_config_init() initializes the config structure with default values.